subprojects/docs/src/samples/userguide/dependencyManagement/workingWithDependencies/iterateDependencies/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+repositories {
+    jcenter()
+}
+
+val scm by configurations.creating
+
+dependencies {
+    scm("org.eclipse.jgit:org.eclipse.jgit:4.9.2.201712150930-r")
+    scm("commons-codec:commons-codec:1.7")
+}
+
+// tag::iteration-task[]
+tasks.create("iterateDeclaredDependencies") {
+    doLast {
+        val dependencySet = configurations.getByName("scm").dependencies
```

Comment:
:x: `scm` is already in scope, this can be simplified to `val dependencySet = scm.dependencies`